Documentation/Input/Pointers.md

```diff
@@ -1,39 +1,239 @@
+![Pointer](../../Documentation/Images/Pointers/MRTK_Pointer_Main.png)
 
-<img src="../../Documentation/Images/Input/PointerProfile.png" style="max-width:100%;">
+This article discusses how to configure and respond to Pointer input in practice, compared to [Pointer Architecture](../Architecture/InputSystem/ControllersPointersAndFocus.md)
 
-MRTK provides a set of pointer prefabs in *Assets/MixedRealityToolkit.SDK/Features/UX/Prefabs/Pointers*. You can use your own prefabs as long as they contain one of the pointer scripts in *Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/Pointers* or any other script implementing [`IMixedRealityPointer`](xref:Microsoft.MixedReality.Toolkit.Input.IMixedRealityPointer).
+Pointers are instanced automatically at runtime when a new controller is detected. More than one pointer can be attached to a controller. For example, with the default pointer profile, Windows Mixed Reality controllers get both a line and a parabolic pointer for normal selection and teleportation respectively.
 
-## Events
+## Pointer configuration
 
-These are all the pointer-related events raised by MRTK:
+Pointers are configured as part of the Input System in MRTK via a [`MixedRealityPointerProfile`](xref:Microsoft.MixedReality.Toolkit.Input.MixedRealityPointerProfile). This type of profile is assigned to a [`MixedRealityInputSystemProfile`](xref:Microsoft.MixedReality.Toolkit.Input.MixedRealityInputSystemProfile) in the MRTK Configuration inspector. The Pointer profile determines the cursor, types of Pointers available at runtime, and how those pointers communicate with each other to decide which one is active.
 
-Event | Description | Handler
---- | --- | ---
-Before Focus Changed / Focus Changed | Raised on both the game object losing focus and the one gaining it every time a pointer changes focus. | [`IMixedRealityFocusChangedHandler`](xref:Microsoft.MixedReality.Toolkit.Input.IMixedRealityFocusChangedHandler)
+- *Pointing Extent* - Defines the max distance for which a Pointer can interact with a GameObject.
+
+- *Pointing Raycast Layer Masks* - This is a prioritized array of LayerMasks to determine what possible GameObjects any given Pointer can interact and the order of interaction to attempt. This is may be useful to ensure Pointers interact with UI elements first before other scene objects.
+
+![Pointer Profile Example](../Images/Input/Pointers/PointerProfile.png)
+
+The default MRTK Pointer Profile configuration includes the following pointer classes and associated prefabs out-of-box. The list of pointers available to the system at runtime is defined under *Pointer Options* in the Pointer profile. Developers can utilize this list to reconfigure existing Pointers, add new Pointers, or delete one.
```

Comment:
>following [](start = 60, length = 9)

This instance of the word following feels like it's a bit out of place, because the text that follows this doesn't contain a list of pointer classes of associated prefabs - maybe this should be "preceeding" because it's referring to the content in the picture?

Reply:
Move image back up to make sense